### ISSUE TITLE
vNext roadmap

### ISSUE THREAD

**Issue by patrickklaeren** [patrickklaeren/AutoRegisterInject#2]
## Tracking issue for vNext

This list represents issues and ideas to be implemented in vNext. Below will not necessarily be released at the same time, but rather over a longer term.

- [x] Add ability to ignore interfaces
- [x] Add ability to auto register hosted services

**Comment by rsousacode** [comment]
I was about to open a issue until I saw this

> Add ability to ignore interfaces 

+1!!!!

When I try to use `RegisterScoped`, basically 100% of the generated code looks like this:

```cs
serviceCollection.AddScoped<System.IDisposable, NewsToVoice.Backoffice.Features.WebSources.WebSourcesService>();
serviceCollection.AddScoped<System.IAsyncDisposable, NewsToVoice.Backoffice.Features.WebSources.WebSourcesService>();

```

**Comment by patrickklaeren** [comment]
@rsousacode Thank you for your comment.

This will now be resolved in the next version (1.3) being published tonight.

https://github.com/patrickklaeren/AutoRegisterInject/pull/4

Track progress here.

This will also resolve this issue.

**Comment by BrianBamies** [comment]
Thanks for updating this issue. It's really helped me and my team out. What an ace repo
